decidim/decidim
# User role is defined for digest notifications to scope translations correctly

#### :tophat: What? Why?
There was a bug  where user role was not given correctly (affected_user or follower) for the digest notification which notifies the user if for example a meeting the user is following / started was closed. Because the user_role scope was missing for the YML -translation files, the email would just give a "translation missing" on the message.

#### Testing
1. Create a proposal in a Decidim instance.
2. Follow the proposal.
3. Answer to the proposal (accepted, evaluating, rejected).
4. Update the notification that was created from the answer in the database and change the "created_at" to 1 day before current date.
5. Run the notifications_digest_daily rake task
6. Check /letter_opener for an email with a "translation missing:" -error message

**Make sure your profile's notification settings is set for daily if you use the daily rake task, and weekly if weekly task used**

### :camera: Screenshots
*Please add screenshots of the changes you are proposing*

**The bug:**

![image](https://user-images.githubusercontent.com/110532525/229720722-b7bd4ee1-eb77-497b-b1e0-8174e28ba98b.png)
:hearts: Thank you!
